README.md

```diff
@@ -15,7 +15,7 @@ Things you may want to cover:
 
 * Database initialization
 
-* How to run the test suite
+* How to run the test suite - bundle exec rake
 
 * Services (job queues, cache servers, search engines, etc.)
 
```

Comment:
I would just go ahead and delete the "default" stuff that is in your Readme. You will need to remove this eventually anyway, so we shouldnt just try to maintain it.